LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/Processors/LnsProtocolMessageProcessor.cs

```diff
@@ -188,7 +194,9 @@ internal async Task InternalHandleDataAsync(RouteValueDictionary routeValues, We
                                                   string json,
                                                   CancellationToken cancellationToken)
         {
-            switch (LnsData.MessageTypeReader.Read(json))
+            var messageType = LnsData.MessageTypeReader.Read(json);
+            var stopwatch = Stopwatch.StartNew();
```

Comment:
We now have a stopwatch here and another one in the DefaultloRaProcessinghandler loop. Maybe we could remove it from the handler and remove it here. **Nothing to do now**, just floating the ideas to take opinions

Reply:
This changed with 06209ea163745c0d8ab6044e390e728c78689c78